ci/tasks/check-mobile-schema-compatibility.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+curl -fsS https://raw.githubusercontent.com/GaloyMoney/galoy-mobile/main/app/graphql/generated.gql -o main.gql
+
+echo Checking compatibility with current main branch
+npx @graphql-inspector/cli validate ./main.gql repo/src/graphql/main/schema.graphql --apollo --noStrictFragments
```

Comment:
it should not be tied to main branch on the backend. it should be running in the current branch.

Reply:
it is the current branch. Its main branch of Galoy-mobile